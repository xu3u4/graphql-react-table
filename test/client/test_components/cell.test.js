import React from 'react';
import renderer from 'react-test-renderer';

import Cell from 'components/common/cell';

describe('Render <Cell>', () => {
  const value = "Jocelyn";
  const cell = renderer.create(
    <Cell>{ value }</Cell>
  );
  
  it('should render <td> with children', () => {
    expect(cell).toMatchSnapshot();
  });
});
