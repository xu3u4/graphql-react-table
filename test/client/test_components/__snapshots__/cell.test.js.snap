// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <Cell> should render <td> with children 1`] = `
<td>
  Jocelyn
</td>
`;
